src/experimental-layers/src/text-layer/text-layer.js

```diff
@@ -63,7 +63,7 @@ export default class TextLayer extends CompositeLayer {
   }
 
   updateState({props, oldProps, changeFlags}) {
-    if (changeFlags.dataChanged || changeFlags.updateTriggers.getText) {
+    if (changeFlags.dataChanged || props.getText) {
```

Comment:
Don't change this line.

Reply:
@georgios-uber it breaks without changing, ib added a suggestion. Is that OK?

Reply:
OK